.github/workflows/test/action.yml

```diff
@@ -31,8 +28,23 @@ runs:
     - name: 📦 Install Dependencies
       shell: bash
       run: |
-        flutter pub global activate very_good_cli
-        very_good packages get --recursive
+        flutter pub get
+        flutter pub get -C sweyer_plugin
+
+    - name: 📄 Check Dart formatting
+      shell: bash
+      run: |
+        echo "::add-matcher::${{ github.workspace }}/.github/problem-matchers/dart-format.json"
+        dart format --set-exit-if-changed .
+        echo "::remove-matcher owner=dart-format::"
+
+    - name: 📝 Run Dart Linter
+      shell: bash
+      run: |
+        echo "::add-matcher::${{ github.workspace }}/.github/problem-matchers/flutter-analyze.json"
+        flutter analyze --fatal-infos || echo "Ignoring analyze problems for now" # TODO: Remove the `|| ...`
```

Comment:
Fixing the remaining warnings and notes is going to take some time, I think it's better if we have the annotations anyways and make them fatal later.